docs/section5/Rule5-16.md

```diff
@@ -0,0 +1,41 @@
+
+# Envelope - Rule 5-18  
+
+**Rule ID:** 5-18  
+**Rule Description:** For building area types included in Table G3.1.1-1, vertical fenestration areas for new buildings and additions shall equal that in Table G3.1.1-1 based on the area of gross above-grade walls that separate conditioned spaces and semi-heated spaces from the exterior.  
+**Rule Assertion:** Baseline RMR = expected value  
+**Appendix G Section:** Section G3.1-5(c) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:**  
+
+- Table G3.1-5. Building Envelope, Baseline Building Performance, c. Vertical Fenestration Areas  
+- Table G3.1.1-1  
+
+**Applicability:** All required data elements exist for B_RMR
+**Applicability Checks:** 
+
+1. Building has spaces that are NEW, ADDITION or ALTERATION.  
+
```

Comment:
This rule does not apply to alterations - alterations must be modeled with the same window area as existed prior to retrofit.